Page/wishlist/wishlist.html

```diff
@@ -24,110 +24,109 @@
             <a class="cart-icon" href="/page/logout/logout.html"><i class="fa-solid fa-right-from-bracket"></i></a>
 
         </nav>
-
-        <main>
-            <div class="wishlist-container">
-                <div class="wishlist-heading">My Wishlist</div>
-                <div class="wishlist-card">
-                    <div class="card .price-card">
-                        <div class="product-image">
-                            <img src="https://images.unsplash.com/photo-1612724189298-89d36b10b26d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1025&q=80"
-                                alt="Image" />
-                            <div class="card-badge-dismiss">
-                                <i class="fa-regular fa-circle-xmark"></i>
-                            </div>
-                        </div>
-                        <div class="card-title">
-                            Nike Air Jordan
-                        </div>
-                        <div class="card-text">
-                            These are Nike Air Jordan 1's
-                        </div>
-                        <div class="all-price">
-                            <span class="price-after">
-                                Rs.14,399
-                            </span>
-                            <span class="price-before">
-                                Rs.15,999
-                            </span>
-                            <span class="discount">
-                                (10% OFF)
-                            </span>
+    </header>
+    <main>
+        <div class="wishlist-container">
+            <div class="wishlist-heading">My Wishlist</div>
+            <div class="wishlist-card">
+                <div class="card .price-card">
+                    <div class="product-image">
+                        <img src="https://images.unsplash.com/photo-1612724189298-89d36b10b26d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1025&q=80"
+                            alt="Image" />
```

Comment:
You might consider using Alt Text as Sneakers instead of Image

Reply:
sure